Task_DEV_4/Task_DEV_4/Parallelogram.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+namespace Task_DEV_4
+{
+    class Parallelogram : Square
+    {
+        public double Height { get; set; }
+        public double Angle { get; set; }
+        public double Width { get; set; }
+
+        public Parallelogram(double length, double height) : base(length)
+        {
+            Height = height;
+        }
+
+        public Parallelogram(double length, double angle, double width) : base(length)
+        {
+            Length = length;
```

Comment:
Why do you set Length here if it exists in the base class?

Reply:
Done.